dist/tools/openocd/openocd.sh

```diff
@@ -97,6 +97,10 @@
 # the file (default).
 # Valid values: elf, hex, s19, bin (see OpenOCD manual for more information)
 : ${IMAGE_TYPE:=}
+# Initial target state when using debug, by default a 'halt' request is sent to
+# the target when starting a debug session. 'reset halt' can also be used
+# depending on the type of target.
+: ${OPENOCD_EXTRA_DEBUG_INIT_CMD:=-c 'halt'}
```

Comment:
Some naming remarks, in that case it is not an `EXTRA` command like the other one as it's mandatory, so this `EXTRA` part should be removed.
To match your comment, maybe put more `START_CMD` as the name `INIT` means something with openocd (`reset init`, `init`)

Also I would prefer it defined along the `DBG` variables, but that's personal taste.